subsys/net/l2/openthread/Kconfig.features

```diff
@@ -193,3 +193,7 @@ config OPENTHREAD_SRP_CLIENT
 config OPENTHREAD_SRP_SERVER
 	bool "Enable SRP Server support"
 	select OPENTHREAD_ECDSA
+
+config OPENTHREAD_MAC_SOFTWARE_TX_SECURITY_ENABLE
```

Comment:
This should rather end up in `Kconfig.thread` instead of `Kconfig.features` - the latter covers features enabled by OT switches during the build, not the configuration.